src/electionguard_gui/components/component_base.py

```diff
@@ -7,16 +7,16 @@
 class ComponentBase(ABC):
     """Responsible for common functionality among ell components"""
 
-    db_service: DbService
-    log: EelLogService
+    _db_service: DbService
+    _log: EelLogService
 
     def init(
         self,
         db_service: DbService,
         log_service: EelLogService,
     ) -> None:
-        self.db_service = db_service
-        self.log = log_service
+        self._db_service = db_service
+        self._log = log_service
```

Comment:
Sorry this refactoring caused a lot of churn, but I was getting frustrated with the inconsistency in underscores. Going forward I'm trying to always use underscores unless it's specifically meant to be public